diff --git a/node_modules/@azure/monitor-opentelemetry/dist/index.js b/node_modules/@azure/monitor-opentelemetry/dist/index.js
index 425f6a4..534ff0d 100644
--- a/node_modules/@azure/monitor-opentelemetry/dist/index.js
+++ b/node_modules/@azure/monitor-opentelemetry/dist/index.js
@@ -34,6 +34,7 @@ var coreClient = require('@azure/core-client');
 var instrumentationBunyan = require('@opentelemetry/instrumentation-bunyan');
 var instrumentationWinston = require('@opentelemetry/instrumentation-winston');
 var sdkLogs = require('@opentelemetry/sdk-logs');
+var { serializableErrorReplacer } = require("@makerx/node-winston/serialize-error");
 
 function _interopNamespaceDefault(e) {
     var n = Object.create(null);
@@ -3611,7 +3612,7 @@ class AzureBatchLogRecordProcessor extends sdkLogs.BatchLogRecordProcessor {
         // Ensure nested log attributes are serialized
         for (const [key, value] of Object.entries(logRecord.attributes)) {
             if (typeof value === "object") {
-                logRecord.attributes[key] = JSON.stringify(value);
+                logRecord.attributes[key] = JSON.stringify(value, serializableErrorReplacer);
             }
         }
         super.onEmit(logRecord);
