diff --git a/node_modules/@opentelemetry/sdk-logs/build/src/LogRecord.js b/node_modules/@opentelemetry/sdk-logs/build/src/LogRecord.js
index 20780f2..7f58d14 100644
--- a/node_modules/@opentelemetry/sdk-logs/build/src/LogRecord.js
+++ b/node_modules/@opentelemetry/sdk-logs/build/src/LogRecord.js
@@ -73,6 +73,10 @@ class LogRecord {
         return this.totalAttributesCount - Object.keys(this.attributes).length;
     }
     setAttribute(key, value) {
+        if (value instanceof Error) {
+          const { message, stack, ...rest } = value;
+          value = { message, stack, ...rest };
+        }
         if (this._isLogRecordReadonly()) {
             return this;
         }
