diff --git a/node_modules/typeorm/platform/PlatformTools.js b/node_modules/typeorm/platform/PlatformTools.js
index 98d578b..6b4f785 100644
--- a/node_modules/typeorm/platform/PlatformTools.js
+++ b/node_modules/typeorm/platform/PlatformTools.js
@@ -122,7 +122,10 @@ class PlatformTools {
      * Normalizes given path. Does "path.normalize".
      */
     static pathNormalize(pathStr) {
-        return path.normalize(pathStr);
+        let normalizedPath = path.normalize(pathStr);
+        if (process.platform === "win32")
+            normalizedPath = normalizedPath.replace(/\\/g, "/");
+        return normalizedPath;
     }
     /**
      * Gets file extension. Does "path.extname".
